src/main/app/src/components/Collapse/Collapse.stories.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { action } from '@storybook/addon-actions';
 import { storiesOf } from '@storybook/react';
 
-import Button from '#/src/components/Button';
+import FormButton from '#/src/components/FormButton';
```

Comment:
Vois ehkä samaan syssyyn muuttaa nämä käyttämään nimettyä importtia.
```suggestion
import { FormButton } from '#/src/components/FormButton';
```